### DISCUSSION TITLE
可以增加全球股指的数据吗？

### DISCUSSION THREAD

**Discussion by PanShawn** [Micro-sheep/efinance#117] [Ideas]
我想找SPX的数据，通过stock.get_quote_history(‘SPX’) 返回的是个股SPIRAX-SARCO ENGINEERING PLC的数据，而不是标普500指数的数据

**Comment by Micro-sheep** [comment]
> 我想找SPX的数据，通过stock.get_quote_history(‘SPX’) 返回的是个股SPIRAX-SARCO ENGINEERING PLC的数据，而不是标普500指数的数据

参考这里：https://github.com/Micro-sheep/efinance/issues/30#issuecomment-1164360290

**Reply by PanShawn** [reply]
谢谢，我通过以下方式成功获取SPX数据，不过1分钟数据好像只有最新交易日的数据，之前的分钟历史数据需要设置哪里参数吗？
import efinance as ef
df = ef.stock.get_quote_history('100.SPX', **{'quote_id_mode': True}, klt = 1)


股票名称 股票代码              日期     开盘  ...  振幅  涨跌幅  涨跌额  换手率
0  标普500      SPX  2023-03-30 21:31  4053.27  ...  0.05    0.59   23.89     0.0
1  标普500      SPX  2023-03-30 21:32  4050.90  ...  0.05   -0.03   -1.05     0.0
2  标普500      SPX  2023-03-30 21:33  4051.15  ...  0.08    0.03    1.40     0.0
3  标普500      SPX  2023-03-30 21:34  4051.85  ...  0.05   -0.01   -0.35     0.0

**Reply by Micro-sheep** [reply]
> 谢谢，我通过以下方式成功获取SPX数据，不过1分钟数据好像只有最新交易日的数据，之前的分钟历史数据需要设置哪里参数吗？
> 
> import efinance as ef
> 
> df = ef.stock.get_quote_history('100.SPX', **{'quote_id_mode': True}, klt = 1)
> 
> 
> 
> 
> 
> 股票名称 股票代码              日期     开盘  ...  振幅  涨跌幅  涨跌额  换手率
> 
> 0  标普500      SPX  2023-03-30 21:31  4053.27  ...  0.05    0.59   23.89     0.0
> 
> 1  标普500      SPX  2023-03-30 21:32  4050.90  ...  0.05   -0.03   -1.05     0.0
> 
> 2  标普500      SPX  2023-03-30 21:33  4051.15  ...  0.08    0.03    1.40     0.0
> 
> 3  标普500      SPX  2023-03-30 21:34  4051.85  ...  0.05   -0.01   -0.35     0.0
> 
> 
> 
> 

一分钟的K线只有最新交易日的。
